portfolio/src/main/webapp/index.html

```diff
@@ -167,12 +168,21 @@ <h3>Shpe</h3>
            Click To Change Color 
         </button>
     </div>
+    -->
+    <div class="fetch">
+        <p>Click <a href="/data">here</a> to view content generated by a servlet.</p>
+        <button onclick="getArrayMessages()">
+            click to get fetch request
+        </button>
+        <div ></div>
```

Comment:
Is this div doing anything? Probably should remove, or at least remove the extra space in it.